### PR TITLE
Udeng 912 locale and keyboard are not correctly set 2

### PR DESCRIPTION
**Pull request by sergio-costas**
This patch adds access to .pam_environment and .xinputrc to allow to set language and keyboard in the core-desktop session.

### CODE REVIEW

**Review by kenvandine — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -159,6 +160,14 @@ plugs:
   shutdown: null
   ssh-keys: null
   system-observe: null
+  dot-config-files:
```
[the remark this answers is not in the document]
Comment (kenvandine) [reply snap/snapcraft.yaml:163]: or even better, maybe shell-session-locale-files?

[the remark this answers is not in the document]
Comment (sergio-costas) [reply snap/snapcraft.yaml:163]: Good point.

[the remark this answers is not in the document]
Comment (sergio-costas) [reply snap/snapcraft.yaml]: Done.

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -167,6 +169,14 @@ plugs:
   shutdown: null
   ssh-keys: null
   system-observe: null
+  shell-session-locale-files:
+    interface: personal-files
+    read:
```
Comment (kenvandine) [inline snap/snapcraft.yaml:174]: I think read is included with write access

Comment (sergio-costas) [reply snap/snapcraft.yaml:174]: You are right... fixing it.

Comment (sergio-costas) [reply snap/snapcraft.yaml]: Done.

**Review by kenvandine — APPROVED** [review]
Looks great, I'll post on the forum requesting access to the new interfaces.

**Pull request merged**